src/components/EmojiPicker.vue

```diff
@@ -64,18 +64,28 @@ export default defineComponent({
 </script>
 
 <style lang="scss">
+@import 'vuetify/settings';
+@import '../assets/styles/settings/_colors.scss';
 .emoji-picker {
   border-radius: 8px;
 
   em-emoji-picker {
     width: 264px;
     height: 264px;
     border-radius: 8px;
+    border-color: map-get($adm-colors, 'light-theme-border');
```

Comment:
This section must contain only common styles for both themes. Don't put here styles like: border-color, color and background-color, etc.

Move the `border-color` to `.v-theme--light {...}`